tool/gravity/cli/rpcagent.go

```diff
@@ -366,6 +367,30 @@ func rpcAgentShutdown(env *localenv.LocalEnvironment) error {
 	return trace.Wrap(err)
 }
 
+// rpcAgentStatus requests and writes the gravity agent statuses to stdout.
+func rpcAgentStatus(env *localenv.LocalEnvironment) error {
```

Comment:
How does this method behave / what does it output if there are no deployed agents?

Reply:
Also, what does it output if there's a failure contacting one of the agents / retrieving info from it?

Reply:
Added additional details and added a 5 sec timeout on the status request.
If an agent fail to return a status response, the agent will be displayed as `Offline` and the version column will be empty.